records.json

```
@@ -89,7 +89,9 @@
     "weblog": "weblogbd.vercel.app",
     "whoshub": "frdze.github.io",
     "www": "katorlys.github.io/rweb.site",
-    "yixboost": "naslambers.synology.me"
+    "yixboost": "naslambers.synology.me",
+    "jingguang": "jingguang.pages.dev",
```

Biome (2.5.6)

[error] The key jingguang was already declared.

(lint/suspicious/noDuplicateObjectKeys)